src/einsteinpy/symbolic/tensor.py

```diff
@@ -307,3 +307,14 @@ def symbols(self):
         
         """
         return self.syms
+
+    def function(self, *args):
+        """
+        Returns arrays of numpy like functions rather than sympy expressions
+
```

Comment:
It actually returns a lamdified function, not arrays of functions.
Change docstring accordingly

Reply:
Something like this
```
        """
        Returns lambdified function of symbolic tensors. 
        This means that the returned functions can accept numerical values and return numerical quantities.

        Parameters
        ----------
            *args
                The variable number of arguments accept sympy symbols. 
                The returned function accepts arguments in same order as initially defined in *args. 
                Uses sympy symbols from class attributes `syms` and `variables`(in the same order) if no *args is passed. 
                Leaving *args empty is recommended.

        Returns
        -------
            function
                Lambdified function which accepts and returns numerical quantities.

        """
```